### PR TITLE
Feature/new context menu

### PR DESCRIPTION
**Pull request by gazizova-vlada**
- Added scroll to all levels of the context menu.
- The structure of all context menu views has been changed and nesting has been added.
- Fixed context menu styles.

### CODE REVIEW

File: packages/shared/components/context-menu/ContextMenu.tsx [hunk 1]
```diff
@@ -60,9 +60,11 @@ const ContextMenu = React.forwardRef((props: ContextMenuProps, ref) => {
   const [model, setModel] = React.useState<ContextMenuModel[] | null>(null);
   const [changeView, setChangeView] = React.useState(false);
   const [showMobileMenu, setShowMobileMenu] = React.useState(false);
-  const [onLoad, setOnLoad] = React.useState<
-    undefined | (() => Promise<ContextMenuModel[]>)
+  const [mobileSubMenuItems, setMobileSubMenuItems] = React.useState<
+    ContextMenuModel[] | undefined
```
Comment (TimofeyBoyko) [inline packages/shared/components/context-menu/ContextMenu.tsx:64]: Pass empty array for initial state

File: packages/shared/components/context-menu/ContextMenu.tsx [hunk 2]
```diff
@@ -260,7 +279,10 @@ const ContextMenu = React.forwardRef((props: ContextMenuProps, ref) => {
 
       // fit
       if (top < document.body.scrollTop) {
-        top = document.body.scrollTop;
+        const marginTop = 16;
```
Comment (TimofeyBoyko) [inline packages/shared/components/context-menu/ContextMenu.tsx:282]: Move from code to const and add description

File: packages/shared/components/context-menu/ContextMenu.styled.ts [hunk 3]
```diff
@@ -299,11 +295,18 @@ const StyledContextMenu = styled.div<{
     &:hover {
       cursor: default !important;
     }
+
+    @media ${mobile} {
+      margin-right: 8px !important;
```
Comment (TimofeyBoyko) [inline packages/shared/components/context-menu/ContextMenu.styled.ts:300]: Try remove !important

File: packages/shared/components/context-menu/ContextMenu.tsx (hunk 1 above)
Comment (gazizova-vlada) [reply packages/shared/components/context-menu/ContextMenu.tsx:66]: Fix, add empty array

File: packages/shared/components/context-menu/ContextMenu.tsx (hunk 2 above)
Comment (gazizova-vlada) [reply packages/shared/components/context-menu/ContextMenu.tsx]: fFix, add  to const and add description

File: packages/shared/components/context-menu/ContextMenu.styled.ts (hunk 3 above)
Comment (gazizova-vlada) [reply packages/shared/components/context-menu/ContextMenu.styled.ts]: Fix, this code has been removed

**Review by TimofeyBoyko — APPROVED** [review]

**Pull request merged**